cmd/xsDeploy.go

```diff
@@ -0,0 +1,424 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/SAP/jenkins-library/pkg/command"
```

Comment:
not properly formatted  ...

Reply:
I don't get an update here when applying `go fmt ./...`